docs/hw/bus/hwsq.rst

```diff
@@ -356,10 +356,10 @@ Opcode: 0x5f <e> <v> - ewait #event #value
 The events are:
 
 - 0: FB_PAUSED [see below]
-- 1: CRTC0_VBLANK [:ref:`nv41 <pcrtc-blank>`, :ref:`G80 <pdisplay-blank>`] 
-- 2: CRTC0_HBLANK [:ref:`nv41 <pcrtc-blank>`, :ref:`G80 <pdisplay-blank>`] 
-- 3: CRTC1_VBLANK [:ref:`nv41 <pcrtc-blank>`, :ref:`G80 <pdisplay-blank>`] 
-- 4: CRTC1_HBLANK [:ref:`nv41 <pcrtc-blank>`, :ref:`G80 <pdisplay-blank>`] 
+- 1: HEAD0_VBLANK [:ref:`nv41 <phead-blank>`, :ref:`G80 <pdisplay-blank>`] 
```

Comment:
touchy issue here.

The head thingie on NV4-NV4x was definitely called PCRTC by nvidia, not PHEAD. This means we'll have an inconsistency between G80+ and pre-G80,, but oh well. PCRTC has to stay.